api/app/Policies/PoolPolicy.php

```diff
@@ -192,4 +192,17 @@ public function deleteDraft(User $user, Pool $pool)
         }
         return $user->isAbleTo("delete-team-draftPool", $pool->team);
     }
+
+    /**
+     * Determine whether the user can archive the pool.
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Models\Pool  $pool
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function archive(User $user, Pool $pool)
+    {
+        $pool->loadMissing('team');
+        return $user->isAbleTo("archive-team-pool", $pool->team);
+    }
```

Comment:
This is going to sound really pedantic on my part, but I'm iffy on the use of the word "restore" to refer to un-archiving something. Restore in Laravel tends to be used pretty commonly with undoing deletion, so is probably a word we should be careful with to avoid confusing ourselves down the line 😅

Reply:
>  I'm iffy on the use of the word "restore" 

Yeah, that's fair.  I originally used soft-deletes where it made more sense but not as much now.  Un-archive is super clunky but it matches the frontend so I'll go with that.
Ref: [restore to unarchive](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7399/commits/50621bfbb81e9f99106a97a274ef3386f7f468ff)

Reply:
> restore also uses this policy

Updated the policy name and comment.
Ref: [policy name](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7399/commits/9d5128b04355a1baee9e8ee8efb530c7ab05c2ef)